fearless/Modules/MainTabBar/MainTabBarPresenter.swift

```diff
@@ -1,17 +1,36 @@
 import Foundation
 import UIKit
+import SoraFoundation
 
 final class MainTabBarPresenter {
     weak var view: MainTabBarViewProtocol?
-    var interactor: MainTabBarInteractorInputProtocol!
-    var wireframe: MainTabBarWireframeProtocol!
+    var interactor: MainTabBarInteractorInputProtocol
+    var wireframe: MainTabBarWireframeProtocol
```

Comment:
Properties was initialisation from constructor, let's change to `let` and maybe protect his to `private`?